flow/project.py

```diff
@@ -69,6 +69,7 @@
 from .labels import classlabel
 from .labels import _is_label_func
 from .util import config as flow_config
+from . import render_status
```

Comment:
```suggestion
from .render_status import Renderer as StatusRenderer
```
Since that is the only class we are using from that module.